srcpkgs/swayidle/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'swayidle'
+pkgname=swayidle
+version=1.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config wayland-devel wayland-protocols scdoc git"
```

Comment:
I don't think wayland-devel actually is a hostmakedepend

Reply:
I think I need it for cross-compile. Without it, I had a problem with another package: https://github.com/void-linux/void-packages/pull/7222#issuecomment-454726995

Reply:
Oh, right. I guess it's required for both hostmakedepends and makedepends